conductor/lib/common.py

```diff
@@ -574,14 +573,32 @@ def load_resources_file():
 
 def get_conductor_instance_types(as_dict=False):
     '''
-    Get the list of available instances types from the resources.yml file
+    Get the list of available instances types.
```

Comment:
Now that this function uses our client api, can we please move this function into the `api_client` module, and name it `request_instance_types` (to match the existing api helper functions).

This will also fix the circular import issue you were having (which is usually a telling sign that something is living in the wrong place).